features/metadata.feature

```diff
@@ -35,7 +35,6 @@ Feature: Metadata
     Background:
         Given a metadata starts with an empty object
 
-    @ignore
     Scenario Outline: Adding some data to metadata
```

Comment:
I think @minz1027  meant the case `Store a record of the metadata when the event is done` seems duplicate. Maybe we can remove it/combine it with the other scenario.